.circleci/config.yml

```diff
@@ -343,6 +354,9 @@ workflows:
       - unit_test_browser:
           name: unit_test_browser_dom_events_plugin
           <<: *requires_build_final
+      - unit_test_browser:
+          name: unit_test_browser_gb_tracker_plugin
+          <<: *requires_build_final
```

Comment:
Actually, I removed the browser-specific tests, which should remove/undo a couple of changes here.